src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialCuratedContentPageObject.java

```diff
@@ -0,0 +1,85 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.photo.PhotoAddComponentObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * @ownership Content
+ */
+public class SpecialCuratedContentPageObject extends SpecialPageObject {
+
+  @FindBy(css = "#addItem")
+  private WebElement addItemButton;
+  @FindBy(css = "#save")
+  private WebElement saveButton;
+  @FindBy(css = "#save.ok")
+  private WebElement successfulSaveMark;
+  @FindBy(css = "li.item:last-of-type .item-input")
+  private WebElement lastElementNameInput;
+  @FindBy(css = "li.item:last-of-type .name")
+  private WebElement lastElementLabelInput;
+  @FindBy(css = "li.item:last-of-type span.remove")
+  private WebElement lastElementRemoveIcon;
+  @FindBy(css = "li.item:last-of-type .image")
+  private WebElement lastElementImage;
+  @FindBy(css = "li.item:last-of-type .image[style]")
+  private WebElement lastElementImageFilled;
+  @FindBy(css = "li.item:last-of-type .image.error")
+  private WebElement lastElementImageWithError;
+
+  public SpecialCuratedContentPageObject(WebDriver driver) {
+    super(driver);
+    PageFactory.initElements(driver, this);
```

Comment:
Your parent constructor is doing that for you `WikiBasePageObject::WikiBasePageObject()`